### DISCUSSION TITLE
Welcome to GHunt Discussions!

### DISCUSSION THREAD

**Discussion by mxrch** [mxrch/GHunt#191] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Kenook** [comment]
I just tried using Ghunt and did python him.py (then entered a real email) and it came back with a syntax error on line 58. Would that be my fault or soemthing else?

**Reply by mxrch** [reply]
Hm there is no him.py file in the project :(

**Reply by Kenook** [reply]
I meant hunt.py, but it's still the same issue.

**Reply by JRRMDSG** [reply]
came here to ask a slightly similar question to this. My python skills are beginner level and I'm struggling with the commands.  There isn't a manual either when I attempted to pull one.  Are there plans for a more robust wiki?

Really excited to give this a run because it is highly recommended, but gosh darnit I can't make it do anything. *shakes fist mockingly*

**Comment by manylittlestorms** [comment]
Hello. Is it possible to get a Google ID account by knowing its youtbe channel ID?

Edit. Is it also possible to get the email from the GAIA ID?

**Comment by clemalo** [comment]
Is it necessary for chrome to be the main browser or just having it is fine ? 
I just tried it a couple of times and it keeps giving me errors whenever it gets to the GMaps reviews

**Reply by clemalo** [reply]
Found the issue, it apparently didn't like the fact that I was using my phone data. 
Worked amazing with Wifi

**Comment by ghost** [comment]
I hava an error:

```
Traceback (most recent call last):
  File "C:\Users\e-lun\Downloads\GHunt-master\ghunt.py", line 7, in <module>
    from lib.utils import *
  File "C:\Users\e-lun\Downloads\GHunt-master\lib\utils.py", line 1, in <module>
    import imagehash
ModuleNotFoundError: No module named 'imagehash'
```

**Comment by Bazili028** [comment]
Hello !
Does this tool write a report (in any format file) ?
I did not find the CLI option for it

**Reply by mxrch** [reply]
Hello, nope

**Comment by biba-samurai228** [comment]
File "/home/laptop/check_and_gen.py", line 61
    <title>GHunt/check_and_gen.py at master · mxrch/GHunt · GitHub</title>
                                            ^
SyntaxError: invalid character '·' (U+00B7

I am new to this, what should i do?

**Comment by diehard88** [comment]
SUGGESTION:

this wonderful tool is all about google/gmail.

why not remove  "@gmail.com" as a requirement in a "ghunt email" query?

"ghunt email jsmith" instead of "ghunt email jsmith@gmail.com".

This is already possible when you authenticate to most google services: @gmail.com is not required.
